Dockerfile

```diff
@@ -8,7 +8,9 @@ RUN go mod download
 
 COPY . .
 
-RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o main .
+RUN go test -v ./...
```

Comment:
Why would you want to run the test here? those are already handled in the test job. And for local development you would probably run the `go test` command locally.

Reply:
True, this is more so a method for having reproducibility in local testing but you're right it doesn't need to be in the final image - would be better to have a dedicated test image if I were to go this route.